nomad/worker.go

```diff
@@ -134,8 +134,9 @@ func (w *Worker) run() {
 func (w *Worker) dequeueEvaluation(timeout time.Duration) (*structs.Evaluation, string, bool) {
 	// Setup the request
 	req := structs.EvalDequeueRequest{
```

Comment:
The worker should probably not sit in a hot loop of calling dequeue on this error, e.g. the failure is basically semi-permanent.